packages/providers/fal/avatar-generation/fal_avatar/models/base.py

```
@@ -0,0 +1,223 @@
+        try:
+            result = fal_client.subscribe(
+                self.endpoint,
+                arguments=arguments,
+                with_logs=True,
+            )
+            processing_time = time.time() - start_time
+            return {
+                "success": True,
+                "result": result,
+                "processing_time": processing_time,
+            }
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)